pages/index.js

```diff
@@ -300,22 +300,22 @@ const StyledFiLogOut = styled(FiLogOut)`
   height: 100%;
 `;
 
-const StyledFriendlistLink = styled(Link)`
-  height: ${theme.button.small};
-  min-width: ${theme.button.small};
-  padding: ${theme.spacing.xs};
-  background-color: ${theme.primaryColor};
-  border-color: ${theme.textColor};
-  border-radius: ${theme.borderRadius.medium};
-  border-width: ${theme.borderWidth.medium};
-  border-style: solid;
-  box-shadow: ${theme.box.shadowSmall};
-`;
-
-const StyledFiUser = styled(FiUser)`
-  width: 100%;
-  height: 100%;
-`;
+// const StyledFriendlistLink = styled(Link)`
+//   height: ${theme.button.small};
+//   min-width: ${theme.button.small};
+//   padding: ${theme.spacing.xs};
+//   background-color: ${theme.primaryColor};
+//   border-color: ${theme.textColor};
+//   border-radius: ${theme.borderRadius.medium};
+//   border-width: ${theme.borderWidth.medium};
+//   border-style: solid;
+//   box-shadow: ${theme.box.shadowSmall};
+// `;
+
+// const StyledFiUser = styled(FiUser)`
+//   width: 100%;
+//   height: 100%;
+// `;
```

Comment:
I guess this is for the case of a rollback?

Reply:
It shouldn't be there. I deleted this block and it's not even in my code anymore